package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asl/projects",
-  "version": "14.1.4",
+  "version": "14.1.3",
```

Comment:
Will this work with newer versions already published to npm - should we instead be releasing a newer version with the reverted code?

Reply:
Changed package version as per discussion